### ISSUE TITLE
Remove hardcoded token(s)

### ISSUE THREAD

**Issue by ScDor** [ScDor/Closure#78]
https://github.com/ScDor/Closure/blob/0230361cb9beb19b0cfcade248fd2b2f59d90356/Closure_Front_End/src/components/SearchBar.vue#L56

A hardcoded token is considered bad practice.  (The one committed is likely to not work elsewhere, as it should be unique to the Django instance that had generated it)

Find all hardcoded "secrets", and then either: 
- Replace them with either a placeholder (e.g. `HERE_BE_TOKEN`) 
- Replace them with a script that loads the token from an external file, and add this file to `.gitignore`

**Comment by ilanerukh** [comment]
Eventually, the token shouldn't be saved at all. When user is login it saves the idToken for the user, which will be used for API requests. I'm working about a "student/me" API and when it will be done it can be changed in the FE.

**Comment by ScDor** [comment]
> Eventually, the token shouldn't be saved at all. When user is login it saves the idToken for the user, which will be used for API requests. I'm working about a "student/me" API and when it will be done it can be changed in the FE.

excellent, please refer to this issue when opening PR (write `fixes: #78` in the PR body)

**Comment by nmdanny** [comment]
You should `this.auth.$http`, which is an Axios instance that contains the authorization header(if logged in), and already has the API URL(from .env) as base url so you don't need to concat it.

Here's example usage of it
```javascript
                const res = await this.$http(`/courses`, {
                    params: {
                        course_id: course.course_id,
                        data_year: course.year
                    }
                })
```


(You can of course use `.then` and `.catch` instead of `async`/`await` syntax)
